package.json

```diff
@@ -23,11 +26,18 @@
   "scripts": {
     "test": "mocha",
     "lint": "eslint **/*.js",
-    "all-contributors": "all-contributors"
+    "all-contributors": "all-contributors",
+    "precommit": "lint-staged && npm test"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/cannawen/metric_units_reddit_bot.git"
   },
-  "license": "GPL-3.0"
+  "license": "GPL-3.0",
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
```

Comment:
This actually auto-fixes whatever it can during a precommit which is really nice!